<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedForm</class>
 <widget class="QWidget" name="FeedForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Feed</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblTitle">
      <property name="text">
       <string>Title: </string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="lblItems">
      <property name="text">
       <string>Items: </string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="lblLanguage">
      <property name="text">
       <string>Language: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QTextEdit" name="txtFeedItems">
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblVersion">
      <property name="text">
       <string>Version: </string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="5">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QLabel" name="lblDescription">
      <property name="text">
       <string>Description: </string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QLabel" name="lblLink">
      <property name="text">
       <string>Link: </string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnExport">
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnSortByTitle">
      <property name="text">
       <string>Sort by title</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>gridLayoutWidget</zorder>
  <zorder>btnExport</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
